.github/workflows/release.yml

```
@@ -7,103 +8,63 @@ defaults:
 
 
 env:
-  project-name: BranchValidator
+  runs-on: ubuntu-latest
 
 
 on:
   workflow_dispatch:
     inputs:
       release_type:
```

Change this workflow input to:

```suggestion
      release-type:
```

Make sure that all references to this input are updated as well